tools/export/uvision4.py

```diff
@@ -28,7 +28,7 @@ class Uvision4(Exporter):
     Exporter class for uvision. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision4'
+    NAME = 'uvision'
```

Comment:
```
$ git grep -n -e "\.NAME"
tools/export/exporters.py:177:        if not self.check_supported(self.NAME):
tools/export/exporters.py:182:        self.project.generate(self.NAME, copied=False, fill=False)
tools/export/exporters.py:192:            self.project_name, self.NAME))
tools/export/exporters.py:194:        result = self.project.build(self.NAME)
```
That corresponds to the `progen_gen_file`, and `progen_build` functions. I'm going to go with no it is not used anywhere.